primitives/src/helpers.rs

```
@@ -218,26 +215,30 @@ pub fn read_g1_point_from_bytes_be(g1_bytes_be: &[u8]) -> Result<G1Affine, &str>
     if !point.is_in_correct_subgroup_assuming_on_curve()
         && is_on_curve_g1(&G1Projective::from(point))
     {
-        return Err("point couldn't be created");
+        return Err(KzgError::NotOnCurveError(
+            "point couldn't be created".to_string(),
+        ));
     }
     Ok(point)
 }
 
-fn get_b_twist_curve_coeff() -> Fq2 {
+fn get_b_twist_curve_coeff() -> Result<Fq2, KzgError> {
     let twist_c0 = Fq::from(9);
     let twist_c1 = Fq::from(1);
 
     // this is bTwistCurveCoeff
     let mut twist_curve_coeff = Fq2::new(twist_c0, twist_c1);
-    twist_curve_coeff = *twist_curve_coeff.inverse_in_place().unwrap();
+    twist_curve_coeff = *twist_curve_coeff
+        .inverse_in_place()
+        .ok_or(KzgError::InvalidDenominator)?;
```

why return InvalidDenominator here? Why not just make this function return an Option like inverse_in_place does?
Seems like inverse_in_place() only returns None when `self.is_zero()`. why does that mean InvalidDenominator?

---

The InvalidDenominator error in this context occurs when trying to compute the inverse of an Fq2 element that has no inverse. But this function can never happen because it's a constant operation, but because of the function `inverse_in_place` returns a possible error, it is bubbling it up than silently ignoring.